ui/analytics.js

```diff
@@ -202,6 +202,9 @@ const analytics: Analytics = {
   openUrlEvent: (url: string) => {
     sendGaEvent('Engagement', 'Open-Url', url);
   },
+  trendingAlgorithmEvent: (trendingAlgorithm: string) => {
+    sendGaEvent('Engagement', 'Trending-Algorithm', trendingAlgorithm);
```

Comment:
Is this an `Engagement` event? Or should it be called something else?

Reply:
(I thought I asked above 4 days ago, just found it stuck pending :upside_down_face:)

Reply:
I think an Engagement event category is fine.  We should be able to slice engagement stats by what algo they were seeing